pkg/cni/plugin/plugin.go

```diff
@@ -155,6 +165,109 @@ func enableKmeshControl(client kubernetes.Interface, pod *v1.Pod) error {
 	return nil
 }
 
+/*
+ * there have a containerID and its hash is 64334212, it have 3 ip address in pod
+ * there have 7 record in thie map.
+ *      |containerID        |ip     |index      |||value        |
+ * 1.   |64334214           |0      |0          |||3            |
+ * 2.   |64334214           |0      |1          |||ip1          |
+ * 3.   |64334214           |0      |2          |||ip2          |
+ * 4.   |64334214           |0      |3          |||ip3          |
+ * 5.   |0                  |ip1    |0          |||0            |
+ * 6.   |0                  |ip2    |0          |||0            |
+ * 7.   |0                  |ip3    |0          |||0            |
```

Comment:
I can not fully get this point.

Reply:
The containerID is mainly used for deletion. When the pod is destroyed, the cni is responsible for clearing the map that records the IP address (used to determine whether it is managed by kmesh). However, I cannot obtain the IP addresses of all network addresses in the cni, and can only obtain the containerID information. So I need the mapping between containerID and ip, which is the content of rows 1-4 in this table